cfe_internal/update/update_policy.cf

```diff
@@ -597,4 +610,64 @@ body delete u_tidy
       rmdirs   => "true";
 }
 
+bundle agent install_modules
+{
+  vars:
+    "vendored_dir" string => "$(sys.inputdir)$(const.dirsep)modules$(const.dirsep)packages$(const.dirsep)vendored$(const.dirsep)";
+    "override_dir" string => "$(sys.inputdir)$(const.dirsep)modules$(const.dirsep)packages$(const.dirsep)";
```

Comment:
I am curious, if you pull your policy into /tmp/craigs_policy, and run it from there, will this do the right thing? I suspect it won't. Generally, I try to stay away from using sys.inputdir. It might be better to instead use sys.policy_entry_dir or $(this.promise_dirname)../ Otherwise, you are leveraging templates that are possibly detached from the executing policy.

Reply:
so true, odd that I thought things were working in this regard. Most sense might be what you said, right. Will change and test. :+1:

Reply:
I used lots of `$(const.dirsep)`, felt kinda noisy and hard to see but seemed "right". I suppose only really needed in case of windows?

What about using `../` a few times, would that need a function wrapper to resolve the path?